stacks/api/storage/src/body.ts

```diff
@@ -27,7 +27,12 @@ abstract class __BsonBody {
       parser(req, res, error => {
         if (error) {
           if (error.type == "entity.too.large") {
-            return observer.error(new HttpException(error.message, HttpStatus.PAYLOAD_TOO_LARGE));
+            return observer.error(
+              new HttpException(
+                error.message + ` maximum file size is ${this.options.objectSizeLimit} MB `,
```

Comment:
If error.message is undefined on prod then we should not depend on it at all since it could make confusing messages if it ever works again.

```suggestion
           `maximum object size is ${this.options.objectSizeLimit}Mi`,
```